conversation_reconstruction/dataflow/conversation_reconstruction/construct_utils/utils/insert_utils.py

```diff
@@ -39,30 +36,28 @@ def isheading(line):
     for x in line:
         if x == '=':
            front_cnt += 1
-    for x in line[::-1]: 
+    for x in line[::-1]:
```

Comment:
are you counting the number of '=' characters? len(re.findall('=', line))

Reply:
No only the consecutive '='s at the beginnging and end of the line (as a wikipedia format).

Reply:
but there's no break statement, you could do len(re.find('^=+').group(0))
or something like that.